### PR TITLE
New Packages: qbe, cproc

### PR DESCRIPTION
**Pull request by lanodan**

### CODE REVIEW

File: sys-devel/cproc/cproc-0_pre20220805.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	S="${WORKDIR}/${CPROC_P}"
+	SLOT="0/${PV}"
+
+	KEYWORDS="~amd64 ~arm64 ~riscv"
+fi
+
+DESCRIPTION="C11 compiler using QBE as backend"
+HOMEPAGE="https://sr.ht/~mcf/cproc/"
+LICENSE="ISC"
+SLOT="0"
```
Comment (thesamesam) [inline sys-devel/cproc/cproc-0_pre20220805.ebuild:23]: Which is right? Presumably this one. It overrides the previous definition

File: sys-devel/qbe/qbe-1.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+else
+	SRC_URI="https://c9x.me/compile/release/${P}.tar.xz"
+	# 64-bit RISC-V only
+	KEYWORDS="~amd64 ~arm64 ~riscv"
+fi
+
+DESCRIPTION="Pure-C embeddable compiler backend"
+HOMEPAGE="https://c9x.me/compile/"
+LICENSE="MIT"
+SLOT="0"
```
Comment (thesamesam) [inline sys-devel/qbe/qbe-1.0-r1.ebuild:18]: No subslot here but you have one for cproc? Seems wrong way around

**Comment by lanodan** [comment]
>+DESCRIPTION="Pure-C embeddable compiler backend"
>+HOMEPAGE="https://c9x.me/compile/"
>+LICENSE="MIT"
>+SLOT="0"
>
>No subslot here but you have one for cproc? Seems wrong way around

Well… none makes sense as they aren't libraries, so I'll put `SLOT=0` on both and remove the double-definition on cproc.

**Comment by thesamesam** [comment]
Thanks!

Done with fixups:

commit d362b1c7e6ed7a7b238d62fe69c90230350b269c (HEAD -> master, origin/master, origin/HEAD)
Author: Sam James <sam@gentoo.org>
Date:   Thu Aug 25 03:08:39 2022 +0100

    sys-devel/cproc: style tweaks, EAPI 8, respect CC

    Signed-off-by: Sam James <sam@gentoo.org>

commit ed18bdc4fcc38d8015b4a17b3537444dcf61b5aa
Author: Sam James <sam@gentoo.org>
Date:   Thu Aug 25 03:06:56 2022 +0100

    sys-devel/qbe: style tweaks, EAPI 8, respect CC

    Signed-off-by: Sam James <sam@gentoo.org>

**Comment by lanodan** [comment]
>commit ed18bdc4fcc38d8015b4a17b3537444dcf61b5aa
>Author: Sam James ***@***.***>
>Date:   Thu Aug 25 03:06:56 2022 +0100
>
>    sys-devel/qbe: style tweaks, EAPI 8, respect CC
>
>    Signed-off-by: Sam James ***@***.***>

It seems like you forgot the "respect CC" part in qbe-9999, both files should have the same content.
Otherwise it looks good to me.

**Pull request closed without merging**